src/LoginControl.js

```diff
@@ -27,24 +27,24 @@ export default class LoginControl extends React.Component {
     );
   }
 
-  handleResponse = authorizationResponse => {
-    // Call updateAuthenticationInfo only if authorizationResponse is valid
-    if (this.isAuthResponseValid(authorizationResponse)) {
-      this.props.updateAuthenticationInfo(
-        new AuthenticationInfo(authorizationResponse)
-      );
+  handleResponse = lwaResponse => {
+    // Call updateAuthenticationInfo only if lwaResponse is valid
+    if (this.isLWAResponseValid(lwaResponse)) {
+      this.props.updateAuthenticationInfo(new AuthenticationInfo(lwaResponse));
     }
   };
 
-  isAuthResponseValid(authResponse) {
-    if (authResponse && authResponse.access_token && authResponse.expires_in) {
+  /**
+   * Returns true if LoginWithAmazon response is valid. False, otherwise.
```

Comment:
Use proper documentation format whenever you choose to write documentation.

Reply:
Done.